pyannote/core/annotation.py

```diff
@@ -472,6 +473,108 @@ def crop(self, support: Support, mode: CropMode = 'intersection'):
             else:
                 raise NotImplementedError("unsupported mode: '%s'" % mode)
 
+    def extrude(self, removed: Support, mode: CropMode = 'intersection') \
+            -> 'Annotation':
+        """Removes all segments or parts of segments that
+        overlap the `removed` Timeline argument.
+
+        A simple illustration:
+
+            annotation
+            A |------|    |------|
+            B                  |----------|
+            C |--------------|              |------|
+
+            removed `Timeline`
+              |-------|  |-----------|
+
+            extruded Annotation with mode="intersection"
+            B                        |---|
+            C         |--|                  |------|
+
+            extruded Annotation with mode="loose"
+            C                               |------|
+
+            extruded Annotation with mode="strict"
+            A |------|
+            B                  |----------|
+            C |--------------|              |------|
```

Comment:
Glad I asked those illustrations.

Looking at the illustrations, the output of "loose" looks a bit more strict than "strict", doesn't it? 
I'd rather not invert "loose" and "strict".

Reply:
Well, the logic behind the names is that "loose" removes any segment that "loosely" overlaps the extrusion support,
and "strict" only removes segments that are strictly included in the support. I tried matching the logic of the crop method, but it obviously might induce some confusion.

Are you still sure you want "strict" and "loose" to reinverted (thus, back to "normal")?

Reply:
OK. Let's stick with the current version.